### PR TITLE
3.1-release-notes

### PR DESCRIPTION
**Pull request by resmaa**
**3.1 Release Notes**

Links to be updated

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1064#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 5815fccb1fb5bdf7e5a417f89ff0c97452202722 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1064/librarium_build/1606284395298164736) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/00-release-notes.md [hunk 1]
```diff
@@ -15,6 +15,85 @@ import Tooltip from "shared/components/ui/Tooltip";
...
+* Palette is now compliant with FIPS compliance and provides a [FIPS-compliant]() version of Kubernetes (PXK and PXK-E).  Palette FIPS support is extended at the Platform Level with the Tenant and Project Scope and Cluster level with FIPS compliant Infra layer cluster profiles.
+* Palette supports tagging and the ability to filter user [access]() and [visibility]() to clusters using tags. 
+* Filter geographically dispersed clusters in the Palette map view and list view using [flexible filters]() to have a granular view of cluster information.
+* Palette supports App Profile versioning. Versioning enables users to create multiple [versions of an App Profile]() within the scope of a single profile name. 
+* Palette supports the [cloning]() of App Profiles across multiple projects. For example, you can clone an app profile created under a specific project to another project within the same tenant. 
+* Palette supports the ability to update an [App Profile]() with the ability to update the versions and manifests. You can also remove and add app tiers.
+* Palette App Mode now supports the use of [containers](). You can specify a single container image when creating an app profile.
```
Comment (karl-cardenas-coding) [inline content/docs/00-release-notes.md:34]:
```suggestion
* Palette App Mode now supports the use of [containers](). You can specify containers when creating an app profile.
```
I had single image, but we can actually add several containers

File: content/docs/00-release-notes.md [hunk 2]
```diff
@@ -15,6 +15,85 @@ import Tooltip from "shared/components/ui/Tooltip";
...
+* Palette leverages the [Helm and OCI registries]() to configure the store and service packs.
```
Comment (karl-cardenas-coding) [inline content/docs/00-release-notes.md:35]: What is the store in this context?

File: content/docs/00-release-notes.md [hunk 3]
```diff
@@ -15,6 +15,85 @@ import Tooltip from "shared/components/ui/Tooltip";
...
+* Palette App Mode now supports the use of [containers](). You can specify a single container image when creating an app profile.
+* Palette leverages the [Helm and OCI registries]() to configure the store and service packs. 
+* Palette supports [out-of-the-box]() applications support for services such as Kafka, MySQL, MinIO, NATS, etc., for Palette Dev Engine. These services can be specified when creating an App Profile.
+* Palette allows you to [pause and resume]() virtual clusters that are not in use. This adds significant flexibility in managing the operating costs and optimizing resource management for virtual clusters.
+
+### Enhancements:
+
+* [OS patch reboot]() support allows clusters to reboot to apply system updates if required.
+
+* Palette Tencent Clusters now support using [security groups]() for network isolation and improving security.
```
Comment (karl-cardenas-coding) [inline content/docs/00-release-notes.md:43]:
```suggestion
* Palette Tencent Clusters now support using [security groups]() for network isolation and improved security.
```

File: content/docs/00-release-notes.md [hunk 4]
```diff
@@ -15,6 +15,85 @@ import Tooltip from "shared/components/ui/Tooltip";
...
+
+* Improved app profile modeling experience.
```
Comment (karl-cardenas-coding) [inline content/docs/00-release-notes.md:45]: What is app profile modeling? Is there a page we can link to? I'm not even sure I know what it is.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]

File: content/docs/00-release-notes.md (hunk 2 above)
Comment (resmaa) [reply content/docs/00-release-notes.md:35]: * Palette leverages the [Helm and OCI registries]() to configure, store and service packs.

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1064#pullrequestreview-1229682869" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1064#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**